src/Jackett.Common/Indexers/BakaBT.cs

```diff
@@ -81,6 +82,18 @@ private async Task DoLogin()
                 { "returnto", "/index.php" }
             };
 
+            //BakaBT for some reason, has two different login pages.
+            try
+            {
+                var loginKeyMatch = Regex.Match(loginForm.ContentString, @"name=\""loginKey\"".+value=\""(.+?)\""");
+                var loginKey = loginKeyMatch.Groups[1].Value;
+                pairs["loginKey"] = loginKey;
+            }
+            catch (Exception)
+            {
+                //Skip
```

Comment:
Don't use try/catch when it's not necessary. It can hided errors if you don't throw or log the exception.